components/etherlink-l2-side/test/ERC20Wrapper.t.sol

```diff
@@ -8,25 +8,29 @@ import {BridgePrecompile} from "../src/BridgePrecompile.sol";
 contract ERC20WrapperTest is Test, IERC20Wrapper {
     ERC20Wrapper public token;
     BridgePrecompile public bridge;
-    address public alice;
-    address public bob;
+    address public alice = vm.addr(0x1);
+    address public bob = vm.addr(0x2);
     uint256 public tokenHash;
+    bytes20 public ticketer = bytes20("some ticketer");
+    bytes20 public wrongTicketer = bytes20("some other ticketer");
+    bytes32 public identifier = keccak256(abi.encodePacked("forged identifier"));
```

Comment:
hashing is not needed here, right?

Reply:
Ah, now I've got it, at first I misunderstood what forged means and thought it requires hashing at the final stage. But right, it makes a lot more sense to provide identifier without hashing and then it can have any length: [34a9ef8](https://github.com/baking-bad/ticketer-proto/pull/1/commits/34a9ef82d2c92a5799131664e74f189f4cc0a145)